dev/fattest.simplicity/src/componenttest/topology/utils/HttpRequest.java

```diff
@@ -234,6 +235,11 @@ public <T> T run(Class<T> type) throws Exception {
                 Log.info(c, "run", "Sending HTTP Request: " + request.getName() + " " + url);
             }
             httpClient.executeMethod(request);
+
+            for (org.apache.commons.httpclient.Header header : request.getResponseHeaders()) {
+                responseHeaders.put(header.getName(), header.getValue());
+            }
+
```

Comment:
Can you update the copyright date on this file please (the 2024 should be replaced with 2025)